md/admin-application-task-list.md

```diff
@@ -0,0 +1,72 @@
+# Task list in Bonita Administrator Application
+
+This page explains what a user with the _Administrator_ in the Bonita Administrator Application can see and do about tasks and more generally, flow nodes.  
+A flow node is an element of a process. It defines an action to be performed (automatic or human). A taks is a flownode done by a person. 
+
+Such users can view the list of flow nodes in error, pending tasks, done tasks, assign and unassign a task, do a task for someone else, skip a failed flow node, replay a failed flow node, and write a comment about the case.
```

Comment:
I don't like the "Such" which seems very formal. But it is personal. I saw we use "those users" at some other places.
```suggestion
Those users can view the list of flow nodes in error, pending tasks, done tasks, assign and unassign a task, do a task for someone else, skip a failed flow node, replay a failed flow node, and write a comment about the case.
```